pkg/client/client.go

```diff
@@ -215,6 +219,23 @@ func (o *OptimizelyClient) GetFeatureVariableString(featureKey, variableKey stri
 	return value, err
 }
 
+// GetFeatureVariableJSON returns the feature variable value of type json associated with the given feature and variable keys.
+func (o *OptimizelyClient) GetFeatureVariableJSON(featureKey, variableKey string, userContext entities.UserContext) (value *optimizelyjson.OptimizelyJSON, err error) {
+
+	val, valueType, err := o.GetFeatureVariable(featureKey, variableKey, userContext)
+	if err != nil {
+		return value, err
+	}
+
+	var convertedValue map[string]interface{}
+	err = json.Unmarshal([]byte(val), &convertedValue)
```

Comment:
We're swallowing this `err` here. That said we don't have to `Unmarshal` here. Why not just pass the to the optimizelyjson `string` constructor and let that object do it's job?

Reply:
we don't swallow this error, we still check for this error and return the same message as for other variables. However, this thing will be rewritten to pass a string as the parameter.